app/server/utils/ussd/kenya_ussd_state_machine.py

```diff
@@ -442,6 +447,20 @@ def process_account_creation_request(self, user_input):
             self.send_sms(self.user.phone, "account_creation_error_sms")
             raise Exception('Account creation failed. Error: ', e)
 
+    def has_default_limit(self, user_input):
+        default_limit = TokenProcessor.get_default_limit(self.user,
+                                                         default_token(self.user),
+                                                         default_transfer_account(self.user))
+        if default_limit:
```

Comment:
`return default_limit != None` ?

Catches possible regression since there is no necessary guarantee that an invalid `default_limit`  must evaluate to `False`.